LondonBritishLibrary/orient/BLorient548.xml

```diff
@@ -31,72 +32,199 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<textLang mainLang="gez"/>
+
+<msItem xml:id="ms_i1.1">
+<locus from="7r"/>
+<title xml:lang="en" type="complete" ref="LIT1716Kidanz">Daily prayer</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="11r"/>
+<title xml:lang="gez" type="complete" ref="LIT1716Kidanz">ሊጦን፡ ዘሰርክ፡</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="14v"/>
+<title xml:lang="en" type="complete">Other prayers entitled <foreign xml:lang="gez">ሊጦን፡</foreign></title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="20r"/>
+<title xml:lang="gez" type="complete" ref="LIT2605Zayena">ዘይነግሥ፡ ዘያሬድ፡</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.5">
+<locus from="23v"/>
+<title xml:lang="gez" type="complete">Eucharistic prayers</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="25r"/>
+<title xml:lang="gez" type="complete">Ordinary canon of the mass</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3">
+<locus from="57r"/>
+<title xml:lang="gez" type="complete" ref="LIT3189Anapho"/>
+<textLang mainLang="gez"/>
+
+
+<msItem xml:id="ms_i3.1">
+<locus from="57r"/>
+<title xml:lang="en" type="complete" ref="LIT1104Anapho">The anaphora of the apostles</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.2">
+<locus from="69r"/>
+<title xml:lang="en" type="complete" ref="LIT1099Anapho">The anaphora of the blessed Virgin Mary, or of Cyriacus</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.3">
+<locus from="78r"/>
+<title xml:lang="en" type="complete" ref="LIT1570habeka">The anaphora of S. John the Evangelist</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.4">
+<locus from="86r"/>
+<title xml:lang="en" type="complete" ref="LIT1103Anapho">The anaphora of the 318 Nicene fathers</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.5">
+<locus from="93r"/>
+<title xml:lang="en" type="complete" ref="LIT1162Athana">The anaphora of Athanasius</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.6">
+<locus from="103r"/>
+<title xml:lang="en" type="complete" ref="LIT1098Anapho">The anaphora of Basil</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.7">
+<locus from="110v"/>
+<title xml:lang="en" type="complete" ref="LIT2040Naakkw">The anaphora of Gregory, the brother of Basil</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.8">
+<locus from="115r"/>
+<title xml:lang="en" type="complete" ref="LIT1347Epipha">The anaphora of Epiphanius</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.9">
+<locus from="120v"/>
+<title xml:lang="en" type="incomplete" ref="LIT3120Anapho">The anaphora of John Chrysostom</title>
+<textLang mainLang="gez"/>
+<note>Imperfect at the end.</note>
+</msItem>
```

Comment:
Are they not part of the Missal?

Reply:
I looked at other missals, so you're right of course, I've rearranged the numbering.